src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/transformer/cas1/Cas1SpaceBookingTransformerTest.kt

```diff
@@ -283,6 +285,211 @@ class Cas1SpaceBookingTransformerTest {
     }
   }
 
+  @Nested
+  inner class TransformJpaToApiWithDepartureReason {
```

Comment:
this is quite a lot of new test code to test one field. Can we update an existing test for at least one of these (e.g. add in an assertion for departure reason where no parent)